app/controllers/conferences_controller.rb

```diff
@@ -64,6 +64,45 @@ def show
     end
   end
 
+  def schedule
+    respond_to do |format|
+      format.ics do
+        calendar = Icalendar::Calendar.new
+        Conference.all.each do |conf|
```

Comment:
Are you sure you want to grab **all** conferences? Those can be quite a few, including conferences in the past. 
I can see this happening *maybe* for all upcoming conferences (if someone would be interested in all of them - big if), otherwise calendar for a specific conference.

Reply:
Yup, it would be useful to be able to programically get calendar for all the conferences, for example for embedding the info about conferences on other websites.